client/src/components/BoardsNavbar/BoardsNavbar.tsx

```diff
@@ -50,7 +64,11 @@ const BoardsNavbar = ({ boardsTitle }: ITest): JSX.Element => {
       >
         <Toolbar>
           <Typography variant="h6" noWrap className={classes.title}>
+<<<<<<< HEAD
```

Comment:
These shouldn't be in the changes. These are conflicts you need to address. Thank you!